package/MDAnalysis/lib/distances.py

```diff
@@ -97,8 +145,15 @@
                                           package="MDAnalysis.lib")
 except ImportError:
     pass
-del importlib
 
+if HAS_DISTOPIA:
+    try:
+        _distances["distopia"] = importlib.import_module(
```

Comment:
If you've already verified that the module is available you don't need to wrap it around a try/except?